File: ElasticErrorRates.Core/Models/LogSummary.cs
namespace ElasticErrorRates.Core.Models
{
    public class LogSummary
    {
        public long HttpUrlCount { get; set; }
        public string HttpUrl { get; set; }
        public string FirstOccurrence { get; set; }
        public string LastOccurrence { get; set; }
        public string Highlight { get; set; }
    }
}

File: ElasticErrorRates.UI/src/app/_shared/api/log/model/logsummary.ts
export interface ILogSummary {
    Id: number;
    HttpUrlCount: string;
    HttpUrl: string;
    FirstOccurrence: Date;
    LastOccurrence: Date;
}

export class LogSummary implements ILogSummary {
    Id: number;
    HttpUrlCount: string;
    HttpUrl: string;
    FirstOccurrence: Date;
    LastOccurrence: Date;
}
